src/components/Header.js

```diff
@@ -89,6 +90,8 @@ class Header extends Component {
       title = titleOfStory
```

Comment:
Perhaps turn this into a funtion that uses a switch (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) with returns.

Reply:
I'll do that.